pkg/apis/extensions/types.go

```diff
@@ -482,6 +471,23 @@ type DaemonSetStatus struct {
 	// ObservedGeneration is the most recent generation observed by the daemon set controller.
 	// +optional
 	ObservedGeneration int64
+
+	// UpdatedNumberScheduled is the total number of nodes that are running updated
+	// daemon pod
+	// +optional
+	UpdatedNumberScheduled int32
+
+	// NumberAvailable is the number of nodes that should be running the
+	// daemon pod and have one or more of the daemon pod running and
+	// available (ready for at least minReadySeconds)
+	// +optional
+	NumberAvailable int32
+
+	// NumberUnavailable is the number of nodes that should be running the
+	// daemon pod and have none of the daemon pod running and available
```

Comment:
@janetkuo confused how come a node can have 2 daemon pods from the same DS. The DS has one podspec and can only schedule one pod per node for a given DS spec.